identity-service/sequelize/migrations/20220912160855-abuse-columns.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn('Users', 'isBlockedFromRelay', {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: false
+      }, { transaction })
+
+      await queryInterface.addColumn('Users', 'isBlockedFromNotifications', {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: false
+      }, { transaction })
+
+      await queryInterface.addColumn('Users', 'appliedRules', {
+        type: Sequelize.JSONB,
+        allowNull: true
+      }, { transaction })
+
+      await queryInterface.sequelize.query(
```

Comment:
discussed IRL: may need to also set isBlockedFromNotifications to true for users who have isBlockedFromRelay = true

Reply:
Did this in the application code instead